src/Symfony/Component/Notifier/Exception/UnsupportedSchemeException.php

```diff
@@ -140,6 +140,10 @@ class UnsupportedSchemeException extends LogicException
             'class' => Bridge\OvhCloud\OvhCloudTransportFactory::class,
             'package' => 'symfony/ovh-cloud-notifier',
         ],
+        'ringcentral' => [
```

Comment:
should be named
```suggestion
        'ring-central' => [
```

Am I right @fabpot?